### PR TITLE
fix(web): fix text out of the box for DNS records

### PR DESCRIPTION
**Pull request by darsene**
ref: DTRSD-117010

| Question         | Answer
| ---------------- | ---
| Branch?          | `master`
| Bug fix?         | yes
| New feature?     | no
| Breaking change? | no
| Tickets          | Fix #DTRSD-117010
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- ~Only FR translations have been updated~
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- ~Breaking change is mentioned in relevant commits~

### CODE REVIEW

File: packages/manager/apps/web/client/app/domain/zone/record/delete/domain-zone-record-delete.html [hunk 1]
```diff
@@ -33,18 +33,23 @@
                         data-translate="domain_configuration_dns_entry_delete_step1_subdomain"
                     ></dt>
                     <dd
+                        class="word-break"
```
Comment (qpavy) [inline packages/manager/apps/web/client/app/domain/zone/record/delete/domain-zone-record-delete.html:36]:
```suggestion
                        class="text-break"
```

File: packages/manager/apps/web/client/app/domain/zone/record/delete/domain-zone-record-delete.html [hunk 2]
```diff
@@ -33,18 +33,23 @@
...
                         data-ng-show="ctrl.entryToDelete.subDomain"
                         data-ng-bind="ctrl.entryToDelete.subDomain + '.' + ctrl.entryToDelete.zoneToDisplay"
                     ></dd>
                     <dd
+                        class="word-break"
```
Comment (qpavy) [inline packages/manager/apps/web/client/app/domain/zone/record/delete/domain-zone-record-delete.html:41]:
```suggestion
                        class="text-break"
```

File: packages/manager/apps/web/client/app/domain/zone/record/delete/domain-zone-record-delete.html [hunk 3]
```diff
@@ -33,18 +33,23 @@
...
                         data-ng-show="!ctrl.entryToDelete.subDomain"
                         data-ng-bind="ctrl.entryToDelete.zoneToDisplay"
                     ></dd>
 
                     <dt
                         data-translate="domain_configuration_dns_entry_delete_step1_target"
                     ></dt>
-                    <dd data-ng-bind="ctrl.entryToDelete.targetToDisplay"></dd>
+                    <dd
+                        class="word-break"
```
Comment (qpavy) [inline packages/manager/apps/web/client/app/domain/zone/record/delete/domain-zone-record-delete.html:50]:
```suggestion
                        class="text-break"
```

File: packages/manager/apps/web/client/app/domain/zone/record/resume.html [hunk 4]
```diff
@@ -8,7 +8,7 @@
         <dd data-ng-bind="ctrl.model.fieldType"></dd>
 
         <dt data-translate="domain_configuration_dns_entry_add_domain"></dt>
-        <dd data-ng-bind="ctrl.getResumeDomain()"></dd>
+        <dd data-ng-bind="ctrl.getResumeDomain()" class="word-break"></dd>
```
Comment (qpavy) [inline packages/manager/apps/web/client/app/domain/zone/record/resume.html:11]:
```suggestion
        <dd data-ng-bind="ctrl.getResumeDomain()" class="text-break"></dd>
```

**Review by qpavy — COMMENTED** [review]
Why not use bootstrap ?

**Review by y4nnL — APPROVED** [review]

**Review by ghyenne — APPROVED** [review]

**Review by anooparveti — APPROVED** [review]

**Pull request merged**